registry/consul/register.go

```diff
@@ -115,14 +130,42 @@ func serviceRegistration(cfg *config.Consul, serviceName string) (*api.AgentServ
 		Address: ip.String(),
 		Port:    port,
 		Tags:    cfg.ServiceTags,
-		Check: &api.AgentServiceCheck{
-			HTTP:                           checkURL,
-			Interval:                       cfg.CheckInterval.String(),
-			Timeout:                        cfg.CheckTimeout.String(),
-			TLSSkipVerify:                  cfg.CheckTLSSkipVerify,
-			DeregisterCriticalServiceAfter: cfg.CheckDeregisterCriticalServiceAfter,
+		// Set the checks for the service.
+		//
+		// If checks > 1 and registry.consul.checksRequired = all, both checks
```

Comment:
According to the reference docs:
```
registry.consul.checksRequired configures how many health checks must pass in order for
fabio to consider a service available.

Possible values are:

one: at least one health check must pass
all: all health checks must pass
```

So I was assuming that in the case _fabio_ HTTP check passes but the TTL doesn't, Consul would mark the service as unhealthy and would not _serve_ this fabio instance to clients, therefore preventing any _blackhole_ of traffic. Now, given you call for attention, I think I got it wrong, because fabio never shares `checksRequired` with Consul, therefore Consul does its own thing regardless of this option value.

Thank you! :)